pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1089,7 +1090,7 @@ func (g *genericScheduler) selectVictimsOnNode(
 	if nodeInfo == nil {
 		return nil, 0, false
 	}
-	potentialVictims := util.SortableList{CompFunc: util.MoreImportantPod}
+	potentialVictims := sortablelist.SortableList{CompFunc: util.MoreImportantPod}
```

Comment:
create a normal slice and use just use ```slice.Sort(potentialVictims, util.MoreImportantPod)``` to sort.

Reply:
util. MoreImportantPod expects an `interface{}` while sort.Slice expects an `int` in its sorting function. Should I change the logic of more important pod to solve this?

Reply:
I have made the change. Can you review it once?